### ISSUE TITLE
It won't start maximized

### ISSUE THREAD

**Issue by ancho85** [doublecmd/doublecmd#1799]
### Version

Double Commander
Version: 1.1.17 gamma
Revision: 548
Commit: f4189f213
Build date: 2024/08/19
Lazarus: 2.2.6.0
FPC: 3.2.2
Platform: x86_64-Linux-gtk2
OS version: Ubuntu 24.04 LTS

Widgetset library: GTK 2.24.33

### Description

Doublecmd won't start maximized. No matter what I do (even with gnome extension to force it). GTK version should remember the last window position after closing. Tried also the QT version and the behavior is the same.
[doublecmd.log](https://github.com/user-attachments/files/16737775/doublecmd.log)
![image](https://github.com/user-attachments/assets/393d3ca1-03d1-4084-9103-29405af05a50)

**Comment by alexx2000** [comment]
Looks like a some GNOME/Wayland incompatibility.

**Comment by ztivolt** [comment]
resize screen manually to almost full, than save the position under config in menu bar.

**Comment by ancho85** [comment]
> resize screen manually to almost full, than save the position under config in menu bar.

Thank you, this workaround works great but if maximized (which happens often) this issue persists. I'll try to remember to leave this window alone in the meantime.

**Comment by ztivolt** [comment]
true, its a workaround.  i tried multiple suggestions as I have the same issue - looks like a common issue with DC - but nothing else worked so far.

**Comment by guy-teube** [comment]
Same here on Ubuntu 24.04.1 64 bits

```
Double Commander
Version: 1.1.31 gamma
Revision: 824
Commit: 49f67723e
Build: 2025/12/21
Lazarus: 4.2.0.0
Free Pascal: 3.2.2
Platform: x86_64-Linux-gtk2
System: Ubuntu 24.04.3 LTS
Desktop Environment: ubuntu (wayland)
Widgetset library: GTK 2.24.33
```

**Comment by ancho85** [comment]
Also, the internal viewer now behaves the same, the workaround won't work on it. I believe this began with version 1.1.32 gamma

**Comment by dweiss** [comment]
> Also, the internal viewer now behaves the same, the workaround won't work on it. I believe this began with version 1.1.32 gamma

Correct - 1.1.31 works for me on ubuntu, later versions reset the viewer window size.
